project.clj

```diff
@@ -4,54 +4,48 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
-  :deploy-repositories [["releases" :clojars]
-                        ["snapshots" :clojars]]
+  :deploy-repositories [["releases" :clojars] ["snapshots" :clojars]]
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
-                 [org.clojure/clojurescript "1.7.122"]
-                 [org.clojure/core.async "0.2.374"]]
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [org.clojure/clojurescript "1.9.293"]]
 
-  :plugins [[lein-cljfmt "0.3.0"]]
+  :plugins [[lein-cljfmt "0.5.6"]]
 
   :source-paths ["src"]
 
-  :clean-targets ^{:protect false} ["resources/public/js/compiled" "target"]
-
-  :cljsbuild {
-    :builds [{:id "dev"
-              :source-paths ["src"]
-
-              :figwheel true
-
-              :compiler {:main reason.core
-                         :asset-path "js/compiled/out"
-                         :output-to "resources/public/js/compiled/reason.js"
-                         :output-dir "resources/public/js/compiled/out"
-                         :source-map-timestamp true }}
-             {:id "min"
-              :source-paths ["src"]
-              :compiler {:output-to "resources/public/js/compiled/reason.js"
-                         :main "reason.core"
-                         :optimizations :advanced
-                         :pretty-print false}}
-             {:id "test"
-              :source-paths ["src" "test"]
-              :compiler {:output-to "target/test.js"
-                         :main "reason.runner"
-                         :optimizations :whitespace
-                         :pretty-print true}}]}
-
-  :profiles {:dev {:dependencies [[cljsbuild "1.1.1"]
-                                  [doo "0.1.6-SNAPSHOT"]
-                                  [figwheel "0.5.0-2"]
-                                  [com.cemerick/piggieback "0.2.1"]
+  :clean-targets ^{:protect false}
+  ["dev-resources/public/js/compiled" "target" "out"]
+
+  :cljsbuild
+  {:builds
+   [{:id "dev"
+     :source-paths ["src"]
+     :figwheel true
+     :compiler
+     {:main reason.core
+      :asset-path "js/compiled/out"
+      :output-dir "dev-resources/public/js/compiled/"
+      :source-map-timestamp true}}
+    {:id "min"
+     :source-paths ["src"]
+     :compiler
+     {:main "reason.core"
+      :optimizations :advanced
+      :output-dir "target/min/"
+      :pretty-print false}}
+    {:id "test"
```

Comment:
we probably want `output-to` in builds (we load reason.js in html but I don't think it will ever be generated)